Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -1647,6 +1647,23 @@ JS_DEFINE_NATIVE_FUNCTION(Window::screen_getter)
     return &impl->screen();
 }
 
+JS_DEFINE_NATIVE_FUNCTION(Window::screen_setter)
```

Comment:
As far as I can tell, nothing about this function is specific to `window.screen`. Can it be moved somewhere where it can be reused for all cases like this?

Reply:
:ninjalinus: